framework/entity/src/main/java/org/apache/ofbiz/entity/model/ModelReader.java

```diff
@@ -225,8 +224,7 @@ private ModelEntity buildEntity(ResourceHandler entityResourceHandler, Element c
                             throw new GenericEntityConfException("Could not get document for " + entityResourceHandler.toString());
                         }
 
-                        // utilTimer.timerString("Before getDocumentElement in " +
```

Comment:
Hi Eugen,

Do we really need this one?

Reply:
I've kept one line comments that seem to be useful in debugging. 
They are not that many so have little impact.